### PR TITLE
Fn ansible

### PR DESCRIPTION
**Pull request by devsuds**

### CODE REVIEW

File: fn_ansible/setup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    name='fn_ansible',
+    version='1.0.0',
+    license='MIT',
+    author='IBM Resilient Labs',
+    author_email='resil.labs@gmail.com',
+    url='https://github.com/ibmresilient/resilient-community-apps',
+    description="Simple agentless IT automation tool.",
+    long_description="Automation scripts are written in YAML files called playbook, which are besically tasks and instructions. Those tasks can be performed on a remote system from resilient using this function.",
+    install_requires=[
+        'resilient_circuits>=30.0.0'
```
Comment (sjcurtinibm) [inline fn_ansible/setup.py]: * Need to add yaml and ansible here with the minimum versions this integration was developed with
* Ansible is currently v2.7.9 on PyPi: https://pypi.org/project/ansible/
* Yaml: what package did you use? Is it on PyPi?

File: fn_ansible/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
+## About Ansible
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/README.md]: Sure. will do that

File: fn_ansible/fn_ansible/components/function_ansible.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/fn_ansible/components/function_ansible.py]: Alright!

File: fn_ansible/fn_ansible/lib/ansible_api.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
```
Comment (sjcurtinibm) [inline fn_ansible/fn_ansible/lib/ansible_api.py]: * Run `pylint` on this file

File: fn_ansible/fn_ansible/lib/ansible_api.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        caution_text = "> That didn't work, please check 3 important things: \
+        \n\t > 1. Configuration in app.config file \n\t \
+        > 2. Playbook variables provided in correct format without extra space \
+        \n\t > 3. Playbook syntax."
+        raise ValueError(original_error + "\n\t" + caution_text)
+    
+    return result
+
+
+def read_secret(file,key):
```
Comment (sjcurtinibm) [inline fn_ansible/fn_ansible/lib/ansible_api.py]: * Avoid using the variable name `file` as it is a predefined variable in Python: https://docs.python.org/2.7/library/functions.html?highlight=file#file

File: fn_ansible/fn_ansible/util/config.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_ansible]
+#playbook_dir=</full/path/to/your/playbook/directory>
```
Comment (sjcurtinibm) [inline fn_ansible/fn_ansible/util/config.py]: * No need to have all these values 'commented out' by default
* We can remove the `#`s

File: fn_ansible/MANIFEST.in [hunk 7]
```diff
@@ -0,0 +1,4 @@
+include README.md
+include fn_ansible/util/*
+include fn_ansible/LICENSE
+include fn_ansible/doc/*
```
Comment (sjcurtinibm) [inline fn_ansible/MANIFEST.in]: * Remove `include fn_ansible/doc/*`
* Add `include screenshots/*`

**Pull request merged**